Add: content about Quota notification (#2471)
getsentry/sentry-docs: src/docs/product/accounts/quotas/index.mdx

## Increasing Quotas

> Add to your quota at any time during your billing period, either by upgrading to a higher tier or by increasing your on-demand capacity. Once on-demand is activated, teammates with the "owner" organization permission level will receive a [notification](/product/alerts-notifications/notifications/#quota) email. While the available plans fit most individual and business needs, Sentry is designed to handle large throughput. If your team needs more, we’re happy to help. Reach out to our sales team at [sales@sentry.io](mailto:sales@sentry.io) to learn more about increasing capacity.

Reviewer: ```suggestion
Add to your quota at any time during your billing period, either by upgrading to a higher tier or by increasing your on-demand capacity. Once on-demand is activated, teammates with the "owner" organization permission level will receive a [notification](/product/alerts-notifications/notifications/#quota) email. Our available plans suit most individual and business needs, and Sentry is designed to handle large throughput. If your team needs more, we’re happy to help. Reach out to our sales team at [sales@sentry.io](mailto:sales@sentry.io) to increase capacity.
```

### How Does Spike Protection Help?

> Because Sentry bills on monthly event volume, spikes can consume your Sentry capacity for the rest of the month. If it really is a spike, spike protection drops events during the spike to try and conserve your capacity. We also send an email [notification](/product/alerts-notifications/notifications/#quota) to the Owner when spike protection is activated.

Reviewer: ```suggestion
Because Sentry bills on monthly event volume, spikes can consume your Sentry capacity for the rest of the month. If it really is a spike, spike protection drops events during the spike to try and conserve your capacity. We also send teammates with the "owner" organization permission level a [notification](/product/alerts-notifications/notifications/#quota) email.
```
